src/DataSets/DataSets.component.js

```diff
@@ -128,6 +132,25 @@ const DataSets = React.createClass({
         });
     },
 
+    getLogs(datasets) {
+        // Set this.state.logs to the logs that include any of the given datasets.
+        if (datasets === null) {
```

Comment:
It's usually safer `if (!datasets)` (like in Python) or even `if (_(datasets).isEmpty())` if an empty list makes no sense as input.